pkg/eventprocessor/impl/executor/impl/executor_test.go

```diff
@@ -0,0 +1,265 @@
+package impl
```

Comment:
You'll see that test files now are organized in different files. We had many of them bundled in the same file and was very hard to track "what was where".

Many of them are the original ones with everything fix to use the new pipeline. 
I'll make some general comments on relevant changes in the wild.